schemas/mod-source-record-manager/jobexecutionlogdeletionpair.schema

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
basically we are not deleting job execution log only, we deleting job execution and all related data.
Also we use camel case in naming convention of schemas.

Reply:
Hi @SerhiiNosko - PR updated as per suggestions, Thanks 🙂